src/Page/CaseStudy.elm

```diff
@@ -0,0 +1,23 @@
+module Page.CaseStudy exposing (view)
+
+import Browser
+import Html.Styled exposing (toUnstyled)
+import I18n.Keys exposing (Key(..))
+import I18n.Translate exposing (translate)
+import Shared exposing (Model, Msg)
+import Theme.PageTemplate as PageTemplate
+
+
+view : Model -> Browser.Document Msg
```

Comment:
The view function for each page should return `Html Msg`

Reply:
duplicate line here too I think.